docs/installation/integrations/react-multiroot.md

```diff
@@ -0,0 +1,139 @@
+
+<info-box hint>
+	The multi-root editors in React is supported since version 6.2.0 of this package.
+
+	Unlike the {@link installation/integrations/react default integration}, in the multi-root editor we prepared the integration based on the hooks and new React mechanisms.
+</info-box>
+
+## Quick start
+
+This guide assumes you already have a React project. If you want to create a new one, you can use the [`create-react-app`](https://create-react-app.dev/) CLI. It allows you to create and customize your project with templates. For example, you can set up your project with TypeScript support.
+
+Install the [CKEditor&nbsp;5 WYSIWYG editor package for React](https://www.npmjs.com/package/@ckeditor/ckeditor5-react) and the {@link installation/getting-started/predefined-builds#multi-root-editor multi-root editor build}. Assuming that you picked [`@ckeditor/ckeditor5-build-multi-root`](https://www.npmjs.com/package/@ckeditor/ckeditor5-build-multi-root):
+
+```bash
+npm install --save @ckeditor/ckeditor5-react @ckeditor/ckeditor5-build-multi-root
+```
+
+Use the `useMultiRootEditor` hook inside your project:
+
+```tsx
+// App.jsx / App.tsx
+
+import React from 'react';
+import { useMultiRootEditor } from '@ckeditor/ckeditor5-react';
+import MultiRootEditor from '@ckeditor/ckeditor5-build-multi-root';
+
+const App = () => {
+	const editorProps = {
+		editor: MultiRootEditor,
+		data: {
+			intro: '<h1>React multi-root editor</h1>',
+			content: '<p>Hello from CKEditor&nbsp;5 multi-root!</p>'
+		},
+		config: {
+			// your config
+		}
+	};
+
+	const {
+		editor, toolbarElement, editableElements,
+		data, setData,
+		attributes, setAttributes
+	} = useMultiRootEditor( editorProps );
+
+
+	return (
+		<div className="App">
+			<h2>Using CKEditor&nbsp;5 multi-root build in React</h2>
+
+			{ toolbarElement }
+
+			{ editableElements }
+		</div>
+	);
+}
+
+export default App;
+```
+
+## Hook properties
+
+The `useMultiRootEditor` hook supports the following properties:
+
+* `editor` (required) &ndash; The {@link module:editor-multi-root/multirooteditor~MultiRootEditor `MultiRootEditor`} constructor to use.
+* `data` &ndash; The initial data for the created editor. See the {@link installation/getting-started/getting-and-setting-data Getting and setting data} guide.
+* `rootsAttributes` &ndash; The initial roots attributes for the created editor.
+* `config` &ndash; The editor configuration. See the {@link installation/getting-started/configuration Configuration} guide.
+* `disabled` &ndash; A Boolean value. The {@link module:editor-multi-root/multirooteditor~MultiRootEditor `MultiRootEditor`} is being switched to read-only mode if the property is set to `true`.
```

Comment:
Could we follow following convention for all parameters of this list?

```javascript
disabled: Boolean
config: EditorConfig
```

etc? WDYT?

Reply:
Instaed of "A Boolean value".